src/components/install-app.tsx

```diff
@@ -203,7 +204,7 @@ export class InstallApp extends React.PureComponent<IProps, IState> {
   }
 
   render() {
-    const appVersionInfo = (window as any).__appVersionInfo;
+    const appVersionInfo = (window as any).__appVersionInfo + " hash: " + serviceWorkerHash;
```

Comment:
Not critical but it would be nice to put something like this in a utils files to DRY it up so it could be used again below in the service worker

```
export const fullVersionInfo = (appVersionInfo: string, serviceWorkerHash?: string) => {
  return `${appVersionInfo}${serviceWorkerHash ? ` hash: ${serviceWorkerHash` : ""}`
}
```